atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/LockWatchValueScopingCacheImpl.java

```diff
@@ -75,7 +76,12 @@ public static LockWatchValueScopingCache create(
                 ResilientLockWatchProxy.newValueCacheProxyFactory(
                         NoOpLockWatchValueScopingCache.create(), metricsManager);
         LockWatchValueScopingCache defaultCache = new LockWatchValueScopingCacheImpl(
-                eventCache, maxCacheSize, validationProbability, watchedTablesFromSchema, proxyFactory::fallback);
+                eventCache,
+                maxCacheSize,
+                validationProbability,
+                watchedTablesFromSchema,
+                proxyFactory::fallback,
+                CacheMetrics.create(metricsManager));
```

Comment:
We could probably add the RLWP metrics to `CacheMetrics`, in which case we could just pass in `CacheMetrics` to this level directly.